docs/plugin-doc.asciidoc.erb

```diff
@@ -44,6 +44,9 @@ This plugin supports the following configuration options:
 <% elsif config[:validate].is_a?(Array) -%>
   * Value can be any of: `<%= config[:validate].join('`, `') %>`
 <% end -%>
+<% if config[:allow_dynamic] -%>
```

Comment:
I'm not sure what the setting should be called, but using `dynamic` to mean `supports sprintf` seems odd. I'd love to have one term for this only.